packages/admin/src/mui/menu/Item.tsx

```diff
@@ -5,26 +5,27 @@ import { IMenuLevel, MenuContext } from "./index";
 import * as sc from "./Item.sc";
 
 export interface IMenuItemProps extends IMenuLevel {
-    text: string;
+    text?: string;
     icon?: React.ReactElement;
     secondaryAction?: React.ReactNode;
     enableMargin?: boolean;
+    children?: React.ReactNode;
```

Comment:
I'm unsure if having text PLUS children for the same thing is a good idea (if we keep text just for compatibility, it should be deprecated)

Reply:
the text property has now been completely removed